integration-tests/src/test/java/org/apache/fineract/integrationtests/BusinessConfigurationApiTest.java

```diff
@@ -96,10 +96,8 @@ public void shouldUpdateStepOrder() {
                 .filter(businessStep -> APPLY_CHARGE_TO_OVERDUE_LOANS.equals(businessStep.getStepName())) //
                 .findFirst().get().getOrder();
         assertEquals(originalOrder + 1, newOrder);
-
-        List<BusinessStep> originalBusinessSteps = getBusinessSteps(originalOrder);
         BusinessStepConfigurationHelper.updateBusinessStepOrder(requestSpec, responseSpec, LOAN_JOB_NAME,
-                BusinessStepConfigurationHelper.toJsonString(originalBusinessSteps));
+                BusinessStepConfigurationHelper.toJsonString(originalStepConfig.getBusinessSteps()));
     }
 
```

Comment:
Please add the following test cases:
1.
- Remove all the steps from the "Loan_job_name"
- Check there is no business steps configured for the loan
- Add "Apply charge to overdue_loans" step
- Check there is 1 business step configured with order 1
- Add "Delinquency Tags business step" to be the first step
- Check there is 2 business steps and the order is correct
- Remove Delinquency Tags business step
- Check there is 1 business step configured with order 1
2. 
- Add a business steps which does not supported by the application -> Correct error flow occurred